_episodes/01-short-introduction-to-Python.md

```diff
@@ -444,16 +444,16 @@ for key in rev.keys():
 ~~~
 {: .language-python}
 ~~~
-1 -> one
-2 -> two
-3 -> three
+'first' -> one
+'second' -> two
+'third' -> three
 ~~~
 {: .output}
 
 > ## Changing dictionaries
 >
 > 1. First, print the value of the `rev` dictionary to the screen.
-> 2. Reassign the value that corresponds to the key `2` so that it no longer
+> 2. Reassign the value that corresponds to the key `second` so that it no longer
 >    reads "two" but instead "apple-sauce".
```

Comment:
```suggestion
>    reads "two" but instead "2".
```

Reply:
I went ahead and added my comment as a suggestion, since you seem on board, @RBenRocks. If you click the "Commit suggestion" box, it'll add my change to your PR. Then I'll merge and we're done!